publish-common/src/main/java/com/ericsson/eiffel/remrem/publish/helper/RabbitMqProperties.java

```diff
@@ -222,6 +233,10 @@ public void init() throws RemRemPublishException {
      */
     public void createRabbitMqConnection() {
         try {
+			if (tcpTimeOut == 0 || tcpTimeOut == null) {
```

Comment:
Shouldn't it be in opposite order, i.e. if (tcpTimeOut == null || tcpTimeOut == 0)? I'm not sure, but there's a risk that tcpTimeOut == 0 throws a NullPointerException if tcpTimeOut is null.